src/devices/quirky.ts

```diff
@@ -0,0 +1,14 @@
+import * as fz from "../converters/fromZigbee";
```

Comment:
Please add to https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/devices/index.ts